### PR TITLE
renovate: Helper action to add change files to renovate PRs

### PR DESCRIPTION
**Pull request by anomiex**
#### Changes proposed in this Pull Request:

When renovate pushes one of its dependency updates, it will be lacking
the required changelogger change files. This action has matticbot add
those files.

#### Jetpack product discussion

None

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* See that the new job is skipped on this PR, as this isn't a renovate PR.
* Merge, then trigger renovate to rebase some of its PRs.

### CODE REVIEW

**Comment by kraftbj** [comment]
If I recall correctly, Renovate will start to ignore a PR once a non-Renovate commit has been pushed to it—so if the dep updates again before we push it, I don't think it'll catch it.

Am I correct in that understanding? What's your take on that UX change?

**Comment by anomiex** [comment]
That's what https://github.com/Automattic/jetpack/pull/19322/files#diff-f82f7d36a61d22f640c25bdb8b0435bf9cb38679d99d5f460753fa668ee7cdb3R9 is doing. See docs at https://docs.renovatebot.com/configuration-options/#gitignoredauthors

Except apparently I typoed it.

File: .github/renovate.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
 	"timezone": "UTC",
 	"schedule": [ "before 3am on the first day of the month" ],
 	"updateNotScheduled": false,
+	"gitIgnoreAuthors": [ "matticbot@users.noreply.github.com" ],
```
Comment (anomiex) [inline .github/renovate.json:9]:
```suggestion
	"gitIgnoredAuthors": [ "matticbot@users.noreply.github.com" ],
```

**Review by kraftbj — APPROVED** [review]

**Comment by anomiex** [comment]
Job ran successfully on a rebased renovate PR: https://github.com/Automattic/jetpack/pull/19328/checks?check_run_id=2219053077

And poking at it on renovate's dashboard did not put it into the "Edited/Blocked" section of the dashboard, so that part seems to be working too.

**Pull request merged**